### ISSUE TITLE
Property 'fields' of exported interface has or is using private name 'Id'

### ISSUE THREAD

**Issue by RaoAkif** [panacloud/cli#126]
I have just added a plain todoSchema, and initialize the project. Right after that run the build command `npm run build`, and the following type errors came.
_Property 'fields' of exported interface has or is using private name 'Id'_

The file named testCollectionsTypes.ts contains the code:
```
import {
  Todo,
  MutationAddTodoArgs,
  MutationUpdateTodoArgs,
  MutationDeleteTodoArgs,
} from "../types";

export type TestCollection = {
  fields: { deleteTodo: { arguments: MutationDeleteTodoArgs; response: Id }[] };
};

```

The errors in command line were

```
mock_lambda_layer/mockData/deleteTodo/testCollectionsTypes.ts:9:72 - error TS2304: Cannot find name 'Id'.

9   fields: { deleteTodo: { arguments: MutationDeleteTodoArgs; response: Id }[] };
                                                                         ~~

mock_lambda_layer/mockData/deleteTodo/testCollectionsTypes.ts:9:72 - error TS4033: Property 'fields' of exported interface has or is using private name 'Id'.

9   fields: { deleteTodo: { arguments: MutationDeleteTodoArgs; response: Id }[] };
```

![image](https://user-images.githubusercontent.com/61361037/147568136-bd011d07-a826-4561-acb3-71d9f1e4868d.png)

**Comment by Muh-Hasan** [comment]
@RaoAkif, Could you please share the schema?

**Comment by RaoAkif** [comment]
> @RaoAkif, Could you please share the schema?

type Todo {
  id: ID!
  title: String!
  done: Boolean!
}

input TodoInput {
  id: ID!
  title: String!
  done: Boolean!
}
input TodoUpdateInput {
  id: ID!
  title: String!
  done: Boolean!
}

type Query {
  getTodos: [Todo]
}

type Mutation {
  addTodo(todo: TodoInput!): Todo
  updateTodo(todo: TodoUpdateInput!): Todo
  deleteTodo(todoId: ID!): ID
}

**Comment by Muh-Hasan** [comment]
Closed by #128
